packages/core/src/domain/session/sessionManager.spec.ts

```diff
@@ -49,11 +54,19 @@ describe('startSessionManager', () => {
 
   function expectSessionIdToBeDefined(sessionManager: SessionManager<FakeTrackingType>) {
     expect(sessionManager.findActiveSession()!.id).toMatch(/^[a-f0-9-]+$/)
+    expect(sessionManager.findActiveSession()?.isExpired).toBeUndefined()
+
     expect(getCookie(SESSION_STORE_KEY)).toMatch(/id=[a-f0-9-]+/)
+    expect(getCookie(SESSION_STORE_KEY)).not.toContain('isExpired=1')
+  }
+
+  function expectSessionToNotBeDefined(sessionManager: SessionManager<FakeTrackingType>) {
+    expect(sessionManager.findActiveSession()).toBeUndefined()
+    expect(getCookie(SESSION_STORE_KEY)).toContain('isExpired=1')
   }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** more accurate naming‏
```suggestion
  function expectSessionToBeExpired(sessionManager: SessionManager<FakeTrackingType>) {
    expect(sessionManager.findActiveSession()).toBeUndefined()
    expect(getCookie(SESSION_STORE_KEY)).toContain('isExpired=1')
  }
```